packages/venia-concept/src/components/MiniCart/section.js

```diff
@@ -1,59 +1,55 @@
-import React, { Component } from 'react';
-import { func, object, oneOf, shape, string } from 'prop-types';
-import Icon from 'src/components/Icon';
+import React from 'react';
+import { bool, func, oneOf, shape, string } from 'prop-types';
 import HeartIcon from 'react-feather/dist/icons/heart';
 import Edit2Icon from 'react-feather/dist/icons/edit-2';
 import TrashIcon from 'react-feather/dist/icons/trash';
-import classify from 'src/classify';
+
+import { mergeClasses } from 'src/classify';
+import Icon from 'src/components/Icon';
+
 import defaultClasses from './section.css';
 
-const SectionIcons = {
+const defaultIconAttributes = {
+    color: 'rgb(var(--venia-teal))',
+    width: '14px',
+    height: '14px'
+};
+const filledIconAttributes = {
+    ...defaultIconAttributes,
+    fill: 'rgb(var(--venia-teal))'
```

Comment:
@jimbo commented on an outdated line so I thought I'd continue the convo here:

Have you tried moving these to a classname and passing them to the Icon component?
```js
// in css
.icon {
  color: rgb(var(--venia-teal));
  width: 14px;
  height: 14px;
}
.iconFilled {
  composes: icon;
  fill: rgb(var(--venia-teal))
}

// in js
const iconClass = isFilled ? classes.iconFilled : classes.icon;
return <Icon classes={classes[iconClass]} />;
```

Reply:
Nvm this won't work as `Icon` requires you pass `attributes` which is different than a css object.